Library/Homebrew/cmd/update.sh

```diff
@@ -9,31 +9,38 @@
 #:      -d, --debug                      Display a trace of all shell commands as they are executed.
 #:      -h, --help                       Show this message.
 
-source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+# HOMEBREW_CURLRC, HOMEBREW_DEVELOPER, HOMEBREW_MISSING_REMOTE_REF_DIRS, HOMEBREW_GIT_EMAIL
```

Comment:
`HOMEBREW_MISSING_REMOTE_REF_DIRS` is now set in `update.sh` so this can perhaps be removed?
```suggestion
# HOMEBREW_CURLRC, HOMEBREW_DEVELOPER, HOMEBREW_GIT_EMAIL
```

Reply:
Yes, the `SC2154` message about `HOMEBREW_MISSING_REMOTE_REF_DIRS` doesn't show when `brew style` with `--enable=all`.
I will remove it.